ipyleaflet/leaflet.py

```diff
@@ -99,6 +112,9 @@ class Marker(UILayer):
     alt = Unicode('').tag(sync=True, o=True)
     rise_on_hover = Bool(False).tag(sync=True, o=True)
     rise_offset = Int(250).tag(sync=True, o=True)
+    rotation_angle = Float(0).tag(sync=True, o=True)
+    rotation_origin = Unicode('').tag(sync=True, o=True)
+    icon = Instance(Icon, allow_none=True, default_value=None).tag(sync=True, o=True, **widget_serialization)
```

Comment:
I think you can remove the `o=True`, because on the JavaScript part you remove `icon` from the options object